tests/filename_map.yml

```diff
@@ -0,0 +1,229 @@
+'*':
```

Comment:
So, I presume that the workflow for adding tests would change so that this file should be modified when appropriate when new tests are added? If so, have we documented that in the documentation on writing Salt tests?